src/main/java/edu/miu/cs545/project/exception/StorageFileNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package edu.miu.cs545.project.exception;
+
+public class StorageFileNotFoundException extends StorageException {
```

Comment:
can this extends  RuntimeException directly?

Reply:
Yes, StorageFileNotFoundException can extend directly from RuntimeException. This depends on how you want to structure your exceptions and what kind of behavior you want them to have. If you want StorageFileNotFoundException to be a more specific type of StorageException, it makes sense to extend from StorageException. However, if you want it to be a standalone runtime exception, you can extend directly from RuntimeException

Reply:
i am OK with both options